### PR TITLE
Fix node version

### PR DESCRIPTION
**Pull request by petrgazarov**
Installs Node version `0.12`. Node `5.7` does not work properly with our setup.

### CODE REVIEW

**Comment by j-clark** [comment]
so, umbrella requires node v 5.4.0 right now.

can we also install n for managing versions? `npm install -g n`

we should also upgrade node in pg-ui at some point

**Comment by bradwheel** [comment]
I vote for installing n.

**Comment by petrgazarov** [comment]
`n stable` currently fetches version `5.6.0`

**Comment by petrgazarov** [comment]
@policygenius/developers :eyeglasses:

**Comment by petrgazarov** [comment]
Minor change, so I'm merging. Lmk any feedback and i will address in another PR.

File: mac.sh [hunk 1]
```diff
@@ -228,7 +228,8 @@ brew_install_or_upgrade 'reattach-to-user-namespace'
 brew_install_or_upgrade 'imagemagick'
 brew_install_or_upgrade 'qt'
 brew_install_or_upgrade 'hub'
-brew_install_or_upgrade 'node'
+brew_install_or_upgrade 'n'
```
Comment (kellywu) [inline mac.sh:231]: nice 👍

**Pull request merged**